### PR TITLE
[MVP-1522] Intercom notification setting view

### PR DESCRIPTION
**Pull request by marukohao**
As a user I can click on settings in the chat window, which lets me edit and see the verification status of my email address, phone number, telegram ID, and browser alert selection.

Test:
updated
Screen Shot 2022-11-11 at 9 33 37 AM

Untitled

Story:
https://notifi.atlassian.net/browse/MVP-1522

### CODE REVIEW

File: packages/notifi-react-card/lib/assets/BackArrow copy.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/assets/BackArrow copy.tsx:1]: I think that this file does not need to be committed - I think that you are using the original back arrow file?

File: packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx [hunk 2]
```diff
@@ -122,6 +124,37 @@ export const IntercomCard: React.FC<
...
+          <div className={'NotifiIntercomCardSettingContent__container'}>
+            <div
+              className={clsx(
+                'NotifiIntercomCard__subtitle2',
+                classNames?.subtitle2,
+              )}
+            >
+              {companySupportDescription}
+            </div>
+            <NotifiIntercomFTUNotificationTargetSection
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx:141]: Can you please add the icons in each target? (ex. Mail icon, cell phone icon, Telegram icon, etc.)

Screen Shot 2022-11-11 at 9 47 51 AM

File: packages/notifi-react-card/lib/components/NotifiEmailInput.tsx [hunk 3]
```diff
@@ -47,9 +55,22 @@ export const NotifiEmailInput: React.FC<NotifiEmailInputProps> = ({
 
   return (
     <>
-      <label className={clsx('NotifiEmailInput__label', classNames?.label)}>
-        {copy?.label}
-      </label>
+      {intercomView ? (
+        intercomCardView.state === 'settingView' ? (
+          <div
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/components/NotifiEmailInput.tsx:60]: Nit: This Resend verification link will appear if in the data, the destination has not been verified. You can take steps to start implementing the logic by passing in a isVerified data and/or setting a setState.

File: packages/notifi-react-card/lib/components/NotifiTelegramInput.tsx [hunk 4]
```diff
@@ -25,14 +27,29 @@ export const NotifiTelegramInput: React.FC<NotifiTelegramInputProps> = ({
...
+    useNotifiSubscriptionContext();
 
   return (
     <>
-      <label className={clsx('NotifiSmsInput__label', classNames?.label)}>
-        {copy?.label}
-      </label>
+      {intercomView ? (
+        intercomCardView.state === 'settingView' ? (
+          <div
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/components/NotifiTelegramInput.tsx:39]: Similar here as well.

Nit: This Resend verification link will appear if in the data, the destination has not been verified. You can take steps to start implementing the logic by passing in a isVerified data and/or setting a setState.

**Review by jennifer-notifi — CHANGES_REQUESTED** [review]
Overall, the components are starting to look really good. Some nit and polish suggestions :)

File: packages/notifi-react-card/lib/assets/BackArrow copy.tsx (hunk 1 above)
Comment (marukohao) [reply packages/notifi-react-card/lib/assets/BackArrow copy.tsx]: as the back arrow icons are not exactly the same(same, color, weight), I renamed the back arrow as IntercomBackArrow.

File: packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx (hunk 2 above)
Comment (marukohao) [reply packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx:141]: attached the updated screenshot in the description box.

File: packages/notifi-react-card/lib/assets/BackArrow copy.tsx (hunk 1 above)
Comment (jennifer-notifi) [reply packages/notifi-react-card/lib/assets/BackArrow copy.tsx]: We could pass in a different color and size but yes, I know that could be hard.

File: packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts [hunk 5]
```diff
@@ -104,6 +105,9 @@ export const useNotifiSubscribe: () => Readonly<{
       setPhoneNumber(phoneNumber ?? '');
 
       const telegramTarget = targetGroup?.telegramTargets[0];
+      if (telegramTarget !== null && emailTarget?.isConfirmed === false) {
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts:108]: Ah, I think you are looking for `if (telegramTarget !== null && telegramTarget?.isConfirmed === false)`.

Actually, a `telegramConfirmationUrl` should not return back if the telegram Id has been confirmed. Instead, in the `NotifiTelegramInput`, you can expose the `telegramConfirmationUrl` and check if it is null or not.

Comment (marukohao) [reply packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts]: double checked with the code in notifi hub, yeah, you are right, we can use telegramConfirmationUrl to check if the telegram has been confirmed or not.

**Review by jennifer-notifi — APPROVED** [review]
Thank you for the updates! It looks good from a UI standpoint, I'm sure we'll make more updates when we add the data APIs :)

**Pull request merged**